sw/device/lib/dif/dif_plic.c

```diff
@@ -16,7 +16,7 @@
 // If either of these static assertions fail, then the assumptions in this DIF
 // implementation should be revisited. In particular, `kPlicTargets`
 // may need updating,
-_Static_assert(RV_PLIC_PARAM_NUMSRC == 84,
+_Static_assert(RV_PLIC_PARAM_NUMSRC == 85,
```

Comment:
This change also require an adjustement inside `sw/device/tests/dif/dif_plic_unittest.cc`. Namely, you'll need to replace `RV_PLIC_IP_2_P_83` by `RV_PLIC_IP_2_P_84`. There are three occurrences on Lines 81, 87, 93.

With this change, CI should pass again. You can test locally using:
```
./meson_init.sh -f
ninja -C build-out all
ninja -C build-out test
```

Reply:
Done, thanks for figuring this out!
`ninja -C build-out all` does not work on my machine, my gcc version is probably too old... This test seems to pass now.

Reply:
No worries. You're right, for `meson/ninja` a different compiler is needed. On the IIS CentOS machines, you can set:
```
export CC=gcc-8.2.0
export CXX=g++-8.2.0
```
In addition, you need some adaptions to config files that I have on my `centos-dev` branch. If you want to try it out, just let me know and I will help you.